.vscode/launch.json

```diff
@@ -0,0 +1,64 @@
+{
+    // Use IntelliSense to learn about possible attributes.
+    // Hover to view descriptions of existing attributes.
+    // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Danswer backend",
+            "type": "python",
+            "request": "launch",
+            "module": "uvicorn",
+            "cwd": "${workspaceFolder}/backend",
+            "env": {
+                "LOG_LEVEL": "DEBUG",
+                "DISABLE_AUTH": "True",
+                "TYPESENSE_API_KEY": "typesense_api_key",
+                "DYNAMIC_CONFIG_DIR_PATH": "./dynamic_config_storage",
+            },
+            "args": [
+                "danswer.main:app",
+                "--reload",
+                "--port",
+                "8080"
+            ]
+        },
+        {
+            "name": "Danswer indexer",
+            "type": "python",
+            "request": "launch",
+            "program": "danswer/background/update.py",
+            "cwd": "${workspaceFolder}/backend",
+            "env": {
+                "LOG_LEVEL": "DEBUG",
+                "PYTHONPATH": ".",
+                "TYPESENSE_API_KEY": "typesense_api_key",
+                "DYNAMIC_CONFIG_DIR_PATH": "./dynamic_config_storage",
+            },
+        },
+        {
+            "name": "Danswer temp files deletion",
+            "type": "python",
+            "request": "launch",
+            "program": "danswer/background/file_deletion.py",
+            "cwd": "${workspaceFolder}/backend",
+            "env": {
+                "LOG_LEVEL": "DEBUG",
+                "PYTHONPATH": "${workspaceFolder}/backend",
+            },
+        },
+        {
+            "name": "Danswer slack bot listener",
+            "type": "python",
+            "request": "launch",
+            "program": "danswer/listeners/slack_listener.py",
+            "cwd": "${workspaceFolder}/backend",
+            "env": {
+                "LOG_LEVEL": "DEBUG",
+                "PYTHONPATH": ".",
+                "DANSWER_BOT_SLACK_APP_TOKEN": "xapp-xxxxxxxxxxxxxxx",
```

Comment:
IMO, these should be set in the `.env` file so we don't need to modify a checked in file to get this running. Can we just make these comments which tell the user to set these in a `.env` file located at the root of the project, and then specify `"envFile": "${workspaceFolder}"`?

Reply:
I agree, this would risk leaking the API keys, 
I think adding an .env.example file in a future PR would be a good idea too.

Reply:
We do have some template `.env` files in the `deployment` directory. Check those out and see if they fit your needs 🙏